frontend/src/index.js

```diff
@@ -19,7 +19,6 @@ ReactDOM.render(
   <ReactKeycloakProvider
     authClient={keycloak}
     initOptions={initOptions}
-    LoadingComponent={<div>Loading</div>}
```

Comment:
We should just keep this for now and at some point make a better loading component

Reply:
reverted the change